runtime/flang/getarg.c

```diff
@@ -27,13 +27,14 @@ __INT_T
 ENTRY(IARGC, iargc)() { return __io_get_argc() - 1; }
 
 void
-ENTRY(GETARGA, getarga)(__INT_T *n, DCHAR(arg) DCLEN64(arg))
+ENTF90(GETARGA, getarga)(__INT_T *n, DCHAR(arg), __INT_T *int_kind DCLEN64(arg))
```

Comment:
Do you really need to change the prefix?

Also, changing the prefix and/or arguments of an existing runtime routine will break backward compatibility. Why not create a new runtime routine (e.g., GETARGA2) that takes care of this case? That way, you will not potentially break someone's existing code.

Or if you prefer, rename this routine to something like GETARGA2 and keep ENTRY(GETARGA) but use it as a wrapper to GETARGA2().